### DISCUSSION TITLE
Skyline emulator is not working on mali GPU and Open game and he stuck not working after update plz solve this and playable on mali GPU plz

### DISCUSSION THREAD

**Discussion by S76042** [skyline-emu/skyline#1802] [General]
Skyline emulator is not working on mali GPU and Open game and he stuck not working after update plz solve this and playable on mali GPU plz

**Comment by s3lverd** [comment]
yes use older builds, newer builds broke mali

**Reply by S76042** [reply]
But fps is so short games lagging

**Reply by s3lverd** [reply]
https://discord.gg/skyline-545842171459272705 join the official skyline emulator discord server for support that would be better

**Reply by S76042** [reply]
My device is snapdragon 680 adreno 610 vulkan drivers link

**Comment by Crytonics** [comment]
Duplicate of #1803 
Don't create multiple Discussions for the same thing tnx.

**Comment by Alkpote81** [comment]
We will always talk about it, as long as the problem remains as it is! There are many people who use gpu mali think about it!

**Reply by Allatel** [reply]
Imagine buying iphone and complaining system is not open source

**Comment by S76042** [comment]
My device is snapdragon 680 adreno 610 turnip masa drivers not working

**Reply by mateoalfaro** [reply]
Normal turnip.is incompatible

**Comment by S76042** [comment]
Skyline emulator vulkan drivers adreno 610 sd 680 link

**Comment by Leaf0007** [comment]
![Screenshot_2022-10-31-19-15-57-685_skyline emu](https://user-images.githubusercontent.com/117083623/199006171-83d0be8f-c626-43d4-8593-fab97ecd46a0.jpg)
 what happen to my device, your gpu is not support?
